db_assessment/optimusprime.py

```diff
@@ -65,11 +65,36 @@ def runMain(args):
     transformersTablesSchemaConfig = {}
     transformersTablesSchemaConfig = transformerConfiguration['tableschemas']
 
-    
+    # SM: 04/28/2022:[Bug#90] If we only intended to run the process for recreating views
+    print('Main with args.recreateviews = {}'.format(str(args.recreateviews).upper()))
+    if str(args.recreateviews).upper() == 'TRUE':
+
+        gcpProjectName = str(args.projectname)
+        bqDataset = str(args.dataset)
+
+        if import_db_assessment.checkDataSetExists(bqDataset, gcpProjectName):
+            
+            transformersParameters['dbversion'] = str(args.dbversion)
+            transformersParameters['optimuscollectionversion'] = str(args.collectionversion)
+            transformersParameters['recreateviews'] = True # setting this as parameter to that we can handle recreation in import_db_assesment.createOptimusPrimeViewsTransformers. If already exists, it returns error by default
+            rulesAlreadyExecuted = [] # keeping it, since it is evaluated in rules engine. although keeping it blank for now as for recreate views this should not impact
+            print(transformersParameters)
```

Comment:
Can we omit this print so we can keep our output cleaner? (our output today is already too long)